.travis.yml

```diff
@@ -0,0 +1,15 @@
+language:
+  node_js
+
+node_js:
+  - "7"
```

Comment:
NB: you can use `./node_modules/.bin/eslint src --fix` to auto-fix simple issues like double-quotes vs single-quotes.